py1

```diff
@@ -0,0 +1,82 @@
+string = '''                   CHAPTER I
+
+                      Down the Rabbit-Hole
+
+
+  Alice was beginning to get very tired of sitting by her sister
+on the bank, and of having nothing to do:  once or twice she had
+peeped into the book her sister was reading, but it had no
+pictures or conversations in it, `and what is the use of a book,'
+thought Alice `without pictures or conversation?'
+
+  So she was considering in her own mind (as well as she could,
+for the hot day made her feel very sleepy and stupid), whether
+the pleasure of making a daisy-chain would be worth the trouble
+of getting up and picking the daisies, when suddenly a White
+Rabbit with pink eyes ran close by her.
+
+  There was nothing so VERY remarkable in that; nor did Alice
+think it so VERY much out of the way to hear the Rabbit say to
+itself, `Oh dear!  Oh dear!  I shall be late!'  (when she thought
+it over afterwards, it occurred to her that she ought to have
+wondered at this, but at the time it all seemed quite natural);
+but when the Rabbit actually TOOK A WATCH OUT OF ITS WAISTCOAT-
+POCKET, and looked at it, and then hurried on, Alice started to
+her feet, for it flashed across her mind that she had never
+before seen a rabbit with either a waistcoat-pocket, or a watch to
+take out of it, and burning with curiosity, she ran across the
+field after it, and fortunately was just in time to see it pop
+down a large rabbit-hole under the hedge.'''
+
+
+
+string
+
+punctuations = '''!"#$%&'()*+, -./:;<=>?@[\]^_`{|}~'''
+for p in punctuations:
+    string = string.replace(p, " ")
+
+print(string)
+
+#create function to count words in text
+
+def word_count(str):
+    counts = dict()
+    words = str.split()
+
+    for word in words:
+        if word in counts:
+            counts[word] += 1
+        else:
+            counts[word] = 1
+
+    return counts
+
+print(word_count(string))
+
+counts = word_count(string)
+print(counts)
+
+import csv
+
+with open('wordcount_test.csv', 'w', newline='') as csvfile: 
+    wordcountwriter = csv.writer(csvfile)
+    fields = ['Word', 'Frequency'] 
+    wordcountwriter.writerow(fields)
+    rows = [['do', '1'],
+           ['once', '1'],
+           ['or', '4'],
+           ['twice', '1']]
+    for row in rows:
+        wordcountwriter.writerow(row)
+
+#create table of word frequency 
```

Comment:
It is customary to leave a space between the # and the comment (some people actually require two spaces, but whatever)